django/conf/locale/sr_Latn/LC_MESSAGES/django.po

```diff
@@ -1,461 +1,581 @@
 # This file is distributed under the same license as the Django package.
-#
+# 
 # Translators:
 # Aleksa Cukovic` <aleksacukovic1@gmail.com>, 2020
 # Danijela Popović, 2022
-# Igor Jerosimić, 2019-2021,2023
+# Igor Jerosimić, 2019-2021,2023-2024
```

Comment:
`django/conf/locale/sr_Latn/LC_MESSAGES/django.mo` should also be changed :thinking:

Reply:
Fetched all missing `.mo` files for `sr_Latn` lang. Thanks for catching this! :trophy: